BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+            file_names[cid] = file_name
+            print("Updated file names:", file_names)  # Debug
+
+            # Save the updated dictionary back to the file
+            with open(FILE_NAMES_FILE, "w") as f:
+                # Pretty-print JSON for readability
+                json.dump(file_names, f, indent=4)
+            print("File saved successfully.")  # Debug
+        except Exception as e:
+            print(f"Error saving file name: {e}")
+
+    def upload_file():
```

Ruff (0.12.2)

Redefinition of unused `upload_file` from line 273

(F811)